bndtools.builder/src/org/bndtools/builder/handlers/baseline/BundleVersionErrorHandler.java

```diff
@@ -23,17 +23,27 @@
 import org.osgi.framework.Constants;
 
 import aQute.bnd.build.Project;
+import aQute.bnd.build.Workspace;
 import aQute.bnd.differ.Baseline.BundleInfo;
 import aQute.bnd.osgi.Builder;
 import aQute.lib.io.IO;
 import aQute.service.reporter.Report.Location;
+import bndtools.central.Central;
 
 public class BundleVersionErrorHandler extends AbstractBuildErrorDetailsHandler {
 
     private static final String PROP_SUGGESTED_VERSION = "suggestedVersion";
 
     private final static String VERSION_ACCEPTING_MACRO_STRING = "(\\d+)\\.(\\d+)\\.(\\d+)\\.([-_\\.\\$\\{\\}\\da-zA-Z]+)";//$NON-NLS-1$
     private final static Pattern VERSION_ACCEPTING_MACRO = Pattern.compile(VERSION_ACCEPTING_MACRO_STRING);
+    private final static LineLocation DUMMY_LINE_LOCATION;
```

Comment:
I don't think it is safe to use a static here. Some other code could modify the LineLocation since it is not immutable. You must create a new LineLocation each time you need it.